RFCs/0020-User-Specified-TestAdapter-Lookup.md

```diff
@@ -0,0 +1,41 @@
+# 0020 User Specified Test Adapter Lookup
+
+## Summary
+This note outlines the proposed changes for lookup and initialization of the test adapters, based on User input.
+
+## Motivation
+
```

Comment:
This is not the motivation.
Let's put this under another heading, "Current behavior".